iceberg/iceberg-handler/src/test/java/org/apache/iceberg/mr/hive/TestHiveIcebergStorageHandlerNoScan.java

```diff
@@ -535,6 +538,38 @@ public void testCreateTableWithoutColumnComments() {
     }
   }
 
+  @Test
+  public void testAlterTableProperties() {
+    TableIdentifier identifier = TableIdentifier.of("default", "customers");
+    shell.executeStatement("CREATE EXTERNAL TABLE customers (" +
+        "t_int INT,  " +
+        "t_string STRING) " +
+        "STORED BY 'org.apache.iceberg.mr.hive.HiveIcebergStorageHandler' " +
+        testTables.locationForCreateTableSQL(identifier) +
+        testTables.propertiesForCreateTableSQL(ImmutableMap.of()));
```

Comment:
why is this part needed here?

Reply:
The location is needed for HadoopTable tests. The properties section is needed to set the catalog type correctly.